app/services/csv_importer.rb

```diff
@@ -15,6 +15,7 @@ def self.import(file)
         check_character: row[3],
         date_account: row[4],
         credit_detail: row[5],
+        case_link: row[6],
```

Comment:
This file is only used for importing new records. It's not related to downloading the CSV